helm-chart/csi-driver/values.yaml

```diff
@@ -0,0 +1,38 @@
+# Secrets:
+# apiToken [Required] - Must be a Linode APIv4 Personal Access Token with all permissions. (https://cloud.linode.com/profile/tokens) 
+apiToken: ""
+
+# region [Required] - Must be a Linode region. (https://api.linode.com/v4/regions)
+region: ""
+
+# Default namespace is "kube-system" but it can be set to another namespace
+namespace: kube-system
+
+# Images - Default 
+csiProvisioner:
+  image: k8s.gcr.io/sig-storage/csi-provisioner
+  tag: v3.0.0
+  pullPolicy: IfNotPresent
+
+csiAttacher:
+  image: k8s.gcr.io/sig-storage/csi-attacher
+  tag: v3.3.0
+  pullPolicy: IfNotPresent
+
+csiResizer:
+  image: k8s.gcr.io/sig-storage/csi-resizer
+  tag: v1.3.0
+  pullPolicy: IfNotPresent
+
+csiLinodePlugin:
+  image: linode/linode-blockstorage-csi-driver 
+  tag: v0.5.0
+  pullPolicy: IfNotPresent
+
+kubectl:
+  image: bitnami/kubectl
+  tag: 1.16.3-debian-10-r36
```

Comment:
Wondering if we should update this older kubectl version to something more recent. We use kubectl to get linode-id and might have to upgrade it sometime in future if it becomes incompatible. The existing one might be working fine and might be ok to use. Not sure about this.

Reply:
Oh I have been running into issue where linode-id isn't matching. I wonder if its because of this. I'll update the version and check